lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/AzureBlockBlobFileSystem.cs

```diff
@@ -157,22 +157,58 @@ public void DeleteDirectory(Uri directoryUri)
         }
```

Comment:
maybe
```C#
Task.WaitAll(directory
    .ListBlobs(true)
    .OfType<ICloudBlob>()
    .Select(blob => blob.DeleteIfExistsAsync())
    .ToArray());
```

Reply:
Parallelization is better. Thanks! Fixed.

---
In reply to: [196651685](https://github.com/apache/reef/pull/1464#discussion_r196651685) [](ancestors = 196651685)